doc/release/3.3.0.rst

```diff
@@ -0,0 +1,375 @@
+Tarantool 3.3
+=============
+
+Release date: November 29, 2024
+
+Releases on GitHub: :tarantool-release:`3.3.0`
+
+The 3.3 release of Tarantool adds the following main product features and improvements for the Community and Enterprise editions:
+
+*   **Community Edition (CE)**
+
+    *   Improvements around queries with offsets.
+    *   Improvement in Raft implementation.
```

Comment:
(Here and same term below) I see 2 ways of capitalization for Raft in the doc: Raft and RAFT. 
https://www.tarantool.io/en/doc/latest/search/?q=raft 
All caps seems more popular -- and "more correct" as for me.

Reply:
I've checked the [Raft website](https://raft.github.io/), [Wikipedia](https://en.wikipedia.org/wiki/Raft_(algorithm)), and some [docs from software vendors](https://developer.hashicorp.com/consul/docs/architecture/consensus), and haven't seen the all caps spelling. Sticking to the original (Raft website) is usually correct.

Reply:
Would be nice to ask our devs to update log messages then: they mostly say "RAFT".